### ISSUE TITLE
Splitting model reducers that are shared

### ISSUE THREAD

**Issue by franferns** [tommikaikkonen/redux-orm#238]
Need suggestions as to how to go about the below issue?

Lets say for example i have 3 models

1. Customer (Contains Many references to Activity and Questionnaire Models)
![image](https://user-images.githubusercontent.com/779253/49268281-ab346c00-f484-11e8-8c02-c666953b2c7a.png)

2. Activity (References Customer)
![image](https://user-images.githubusercontent.com/779253/49268330-e59e0900-f484-11e8-8413-141f3faffe20.png)

3. Questionnaire (References Questionnaire)
![image](https://user-images.githubusercontent.com/779253/49268368-1bdb8880-f485-11e8-8087-0735c4c3f4ba.png)

This works fine when i have a single createReducer but when i try to split Activity and Questionnaire i get the already defined error. The above example is a generalized example. I have around 250+ reducers which i need to split . 

![image](https://user-images.githubusercontent.com/779253/49268748-44648200-f487-11e8-9f8b-8030ecc5f05d.png)

ORM for 1 reducers 
![image](https://user-images.githubusercontent.com/779253/49268598-5691f080-f486-11e8-85dd-9e3051a68964.png)

ORM for 2 reducers
![image](https://user-images.githubusercontent.com/779253/49268608-68739380-f486-11e8-8cd3-aa576ed3a6e9.png)

![image](https://user-images.githubusercontent.com/779253/49268644-9ce74f80-f486-11e8-8974-e2812ab4b2d5.png)

Basically i need to split these so all the reducers not required in a module are not called. How do i go about this any suggestions or have i understood this wrong ?

**Comment by haveyaseen** [comment]
See #215 and #179, this is due to the usage of both `many` and `fk`. Remove the `many` fields in your Customer model, splitting reducers sounds like it's dangerous: re-using the same models for different ORMs is likely to cause bugs.
